aiida/common/profile.py

```diff
@@ -126,6 +126,28 @@ def rmq_prefix(self):
     def is_test_profile(self):
         return self._test_profile
 
+    @property
+    def configured_user_email(self):
```

Comment:
Do we need to introduce a different adjective (*configured* user) here?
Why not *default* user?

Reply:
Changed to `default_user_email`